recipes/librealsense/all/patches/patch-libusb-include.patch

```diff
@@ -0,0 +1,16 @@
+--- CMake/global_config.cmake
++++ CMake/global_config_new.cmake
+@@ -111,10 +111,10 @@
+ 
+ macro(add_tm2)
+     message(STATUS "Building with TM2")
+-    include(libusb_config)
+-    target_link_libraries(${LRS_TARGET} PRIVATE usb)
++    find_package(libusb REQUIRED)
++    target_link_libraries(${LRS_TARGET} PRIVATE libsusb)
+     if(USE_EXTERNAL_USB)
+-        add_dependencies(${LRS_TARGET} libusb)
++        add_dependencies(${LRS_TARGET} libusb)
```

Comment:
Since `libusb` is the `lib.a` there's no cmake target and there's no dependency

Reply:
Indeed, this does nothing. There is no other target to consider/build in the dependency tree.